### PR TITLE
Documentation added for browsing the H2 database

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: en/docs/administer/browsing-the-h2-database.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+1.  Open the
+    `           <IS_HOME>/repository/conf/deployment.toml          `
+    file and add the following configuration.
+
+    ``` toml
+    [database_configuration]
+    enable_h2_console = "true"
+    ```
+
+2.  Restart the WSO2 product and access the following URL via your
```
Comment (YvonneW) [inline en/docs/administer/browsing-the-h2-database.md]: Restart WSO2 Identity Server

File: en/docs/administer/browsing-the-h2-database.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <thead>
+      <tr class="header">
+      <th>Element</th>
+      <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr class="even">
+      <td><strong>username</strong></td>
+      <td>username of the database. By default wso2carbon</td>
```
Comment (YvonneW) [inline en/docs/administer/browsing-the-h2-database.md]: `wso2carbon`

File: en/docs/administer/browsing-the-h2-database.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      </tr>
+      <tr class="even">
+      <td><strong>password</strong></td>
+      <td>password of the database. By default wso2carbon</td>
```
Comment (YvonneW) [inline en/docs/administer/browsing-the-h2-database.md]: `wso2carbon`

File: en/docs/administer/browsing-the-h2-database.md (hunk 2 above)
Comment (Buddhimah) [reply en/docs/administer/browsing-the-h2-database.md]: done

File: en/docs/administer/browsing-the-h2-database.md (hunk 1 above)
Comment (Buddhimah) [reply en/docs/administer/browsing-the-h2-database.md]: done

File: en/docs/administer/browsing-the-h2-database.md (hunk 3 above)
Comment (Buddhimah) [reply en/docs/administer/browsing-the-h2-database.md]: done

**Pull request merged**